x/concentrated-liquidity/pool_test.go

```diff
@@ -257,6 +257,82 @@ func (s *KeeperTestSuite) TestCalculateSpotPrice() {
 	s.Require().True(spotPrice.IsNil())
 }
 
+func (s *KeeperTestSuite) TestGetTotalPoolLiquidity() {
+	var (
+		usdcCoin  = sdk.NewCoin(USDC, sdk.OneInt())
+		ethCoin   = sdk.NewCoin(ETH, sdk.NewInt(2))
+		uosmoCoin = sdk.NewCoin("uosmo", sdk.NewInt(3))
+
+		defaultCoins = sdk.NewCoins(usdcCoin, ethCoin)
+	)
```

Comment:
nit: maybe we can call the underlying assets "defaultPoolCoin1" and "defaultPoolCoin2" with the third coin being "nonPoolCoin". I had to keep looking up to the vars to determine which coin was the "nonPoolCoin" and which were the "poolCoin"

Reply:
Done